breeze

```diff
@@ -1361,9 +1361,9 @@ function breeze::parse_arguments() {
             ;;
         *)
             breeze::usage
-            echo >&2
-            echo >&2 "ERROR: Unknown command ${1}"
-            echo >&2
+            echo
+            echo -e " \e[31mERROR: Unknown command\e[0m"
+            echo
```

Comment:
Why not on stderr anymore?

Reply:
> Terminals, no, but cases when output is not actually a terminal.

Not a problem for CI/dev tools.

Reply:
> Why not on stderr anymore?

The main reason for printing it to stderr is that some tools will colour it differently (red).It used to be standard way when terminals did not support colors, but now when all of them do, it's rather useful for dev tools.

There are no other benefits unless the script/tool is used in Kubernetes, etc. for running in production (I those cases intact) or when the output of script is parsed by another tool (none of those scripts produced parseable outout). 

Those scripts are meant to be run in two cases only: 

1) When user runs in it manually
2) When they run in CI environment

In both cases coloring part of the output manually is better than redirecting to stderr nowadays. All the modern CI systems and all the terminals for developers (including thsoe embedded in cloud tools and IDEs) support coloured output. Similarly as refresh button in Airflow UI, let's move to the XXI century.